README.md

```diff
@@ -1 +1,203 @@
-# Python boilerplate for GitHub tasks
```

Comment:
Add: 
# Linter Formatter
- Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start